Automattic/vip-go-mu-plugins
# E2E: Add testing of Media in pages, posts and directly

## Description

This PR adds checks for adding media to posts/pages as well as a separate test for uploading media directly

## Changelog Description

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Ensure e2e tests pass in CI

To Run Tests locally:
1. Start e2e env `npm run setup-e2e-env`
2. Run Tests `npm run test-e2e`
